FastSimulation/ShowerDevelopment/interface/FastHFShowerLibrary.h

```diff
@@ -45,61 +46,32 @@ class FastHFShowerLibrary {
 
 public:
 
-  struct Hit {
-    Hit() {}
-    G4ThreeVector             position;
-    int                       depth;
-    double                    time;
-  };
-
   void                initRun();
```

Comment:
@Vlandr57 

If you move the parCode condition into HFShowerLibrary::fillHits,
FastHFShowerLibrary::initRun can be removed.